### ISSUE TITLE
关于超声图像

### ISSUE THREAD

**Issue by wangzhenlin123** [zju3dv/MatchAnything#4]
超声匹配对齐是业内难题，请问作者有没有测试过对超声图像的多模态匹配？

**Comment by hxy-123** [comment]
Hi, we have tried our pre-trained model on MRI-Ultrasound image slices but did not include the quantitative evaluations in the paper. Here are some qualitative results. Without seeing these types of modalities in training, our pre-trained model produces plausible results but may contain incorrect matches. For better accuracy and robustness, I think our model can be finetuned for this specific task. You can use our Huggingface online demo to try your data!

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/214f76be-7339-4173-b41f-5436e0914e4d" />

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/27acd119-b864-4a5e-9508-aa432005bb48" />
